tools/rpkg/src/duckdbr.cpp

```diff
@@ -127,6 +127,7 @@ static SEXP cstr_to_charsexp(const char *s) {
 		Rf_error("cpp_str_to_charsexp: Memory allocation failed");
 		UNPROTECT(1);
```

Comment:
`Rf_error()` makes a long jump, no need to `UNPROTECT()` this path.

Reply:
that's a good point